mock-core-configs/etc/mock/amazonlinux-2-aarch64.cfg

```diff
@@ -0,0 +1,25 @@
+config_opts['root'] = 'amzn2-aarch64'
```

Comment:
We usually had two dashes in the name of the root. Following the patter $name-$version-$arch. I would like to see here 'amzn-2-aarch64'. This is not mandatory in mock itself. But I have seen some code based on top of mock, which assumes this schema.

Reply:
@xsuchy, thanks for the feedback. I've made changes to to ensure $name$-version-$arch is enforced as well as using config template.